C_CPP/Data Structures/Linked List/Linked_List_Basic_code_Video.md

```diff
@@ -0,0 +1,2 @@
+Adding video tutorial drive link for understanding linked list basic syntax and code.
```

Comment:
Observe & do the same
https://github.com/girlscript/winter-of-contributing/pull/3407/files

Reply:
I made the required changes. Can you check bhaiya?

Reply:
But I am facing the issue of not getting thumbnail of image. So I tried through google drive but it still does not work.